cadl-tests/cadl/paged.cadl

```diff
@@ -25,6 +25,6 @@ model Resource {
 
 @route("/paged")
 interface PagedOp {
-  @convenienceAPI("ListCONVENIENCE")
+  @convenientAPI("ListCONVENIENCE")
```

Comment:
Does `convenientAPI` still take a string? If not, just remove the rename part.

You can double check with Pan.

Reply:
I checked the code previously, it takes an operation and a boolean value, I am still checking the test to learn a sample, then update our cadl if needed.

Reply:
Operation is just what it decorates. The boolean should be true/false that controls whether to generate.

Make sure that `@convenientAPI(false)` can turn off the convenience API.

Reply:
sure, will check.